torch/csrc/DynamicTypes.cpp

```diff
@@ -100,6 +105,14 @@ at::Type& getATenType(PyTypeObject* type)
   throw std::invalid_argument("unsupported Python type");
 }
 
+THPDtype* getDtype(at::Type& type) {
+  auto it = attype_to_dtype.find(&type);
+  if (it != attype_to_dtype.end()) {
+    return it->second;
+  }
+  throw std::invalid_argument("unsupported at::Type");
```

Comment:
Including the name of the function will help debugging in case this is actually hit. e.g.:

```
getDtype(): unsupported at::Type
```